### DISCUSSION TITLE
我想问下trust4如何可以直接分析10X平台单细胞的.fastq格式双端数据？而不是去分析bam格式

### DISCUSSION THREAD

**Discussion by fight2021** [liulab-dfci/TRUST4#270] [General]
我想问下trust4如何可以直接分析10X平台单细胞的.fastq格式双端数据？而不是去分析bam格式，能否提供相关分析支持，现在分析速度太慢了

**Comment by mourisl** [comment]
Yes, you can. It would be something like running fastq files from this section: https://github.com/liulab-dfci/TRUST4?tab=readme-ov-file#10x-genomics-data-and-barcode-based-single-cell-data . For the running speed, which version of TRUST4 are you using? Which step do you find is too slow?

**Comment by fight2021** [comment]
However, I found that this command can only be used for general transcriptome data and cannot perform immune repertoire data analysis on single-cell transcriptome data. The command is: run-trust4 -f hg38_bcrtcr.fa --ref human_IMGT+C.fa -u path_to_10X_fastqs/R2.fastq.gz --barcode path_to_10X_fastqs/R1.fastq.gz --readFormat bc:0:15 --barcodeWhitelist cellranger_folder/cellranger-cs/VERSION/lib/python/cellranger/barcodes/737K-august-2016.txt [other options].

**Comment by mourisl** [comment]
Duplicated question in issue #271 . Close for now.
